### PR TITLE
GEODE-3629: Old versions will be pulled from the Apache Maven Repo

### PR DESCRIPTION
**Pull request by jhuynh1**
Modified the geode-old-versions build to now pull down the from the maven repository instead of a mirror.  This zip should now be cached locally.

### CODE REVIEW

File: geode-old-versions/build.gradle [hunk 1]
```diff
@@ -15,13 +15,16 @@
...
-  id "de.undercouch.download" version "3.2.0"
-}
 
-import de.undercouch.gradle.tasks.download.Download
 disableMavenPublishing()
 
+repositories {
+  maven { url "https://plugins.gradle.org/m2/" }
+  maven {
+    url 'https://repository.apache.org/content/repositories/snapshots'
```
Comment (upthewaterspout) [inline geode-old-versions/build.gradle]: Why did you add the snapshots repository? It seems like we should be testings against an official old version, not a nightly snapshot.

**Comment by jhuynh1** [comment]
Whoops, I understand the issue now, sorry! I'll remove the repository changes and run precheckin

File: geode-old-versions/build.gradle [hunk 2]
```diff
@@ -15,13 +15,13 @@
...
 
-plugins {
-  id "de.undercouch.download" version "3.2.0"
-}
 
-import de.undercouch.gradle.tasks.download.Download
 disableMavenPublishing()
 
+repositories {
+  maven { url "https://plugins.gradle.org/m2/" }
```
Comment (upthewaterspout) [inline geode-old-versions/build.gradle]: Looks like there is still a repository being added in this diff?
